### ISSUE TITLE
As a reader I want to see all the commencement details to do with a work on its History timeline

### ISSUE THREAD

**Issue by goose-life** [laws-africa/peachjam#1450]
For example, the following work has multiple commencements with notes on the platform:

https://edit.laws.africa/works/akn/za/act/1991/108/
![image](https://github.com/laws-africa/peachjam/assets/32566441/861cfff8-919e-4075-ba7a-5eefbfa4d888)

But only the 'main' commencement is shown on the website:

https://lawlibrary.org.za/akn/za/act/1991/108/eng@1991-06-28
![image](https://github.com/laws-africa/peachjam/assets/32566441/89153a9d-f79e-4545-9b14-a7cda92abb93)

Commencement notes should also be shown when they exist.

When working out the timeline events in https://github.com/laws-africa/peachjam/blob/main/peachjam/views/legislation.py#L161, the commencements list in the metadata_json should be inspected rather than using the commencement date etc that have been recorded on the work. 

In other words, commencements should be treated a lot more like amendments than like a publication or assent date.

One thing to note is that when there's an amendment and a commencement by the same work at the same date, we don't need to display the commencement as well — but we do want to keep its note if there is one.

Also, a new endpoint, …/akn/…/commencements.json, is being added soon, but metadata_json.commencements should do the trick for now. The commencement information will be removed from the serialized work data with v3 of the API.

**Comment by goose-life** [comment]
closing in favour / as duplicate of #996
